DigitalLearningSolutions.Data.Tests/DataServices/SessionDataServiceTests.cs

```diff
@@ -1,151 +1,167 @@
+            {
+                // When
+                const int candidateId = 101;
+                sessionDataService.StopDelegateSession(candidateId);
+
+                // Then
+                var sessions = sessionTestHelper.GetCandidateSessions(candidateId);
+                var activeSessions = sessions.Where(session => session.Active);
+
+                activeSessions.Should().BeEmpty();
+            }
+        }
+
+        [Test]
+        public void UpdateDelegateSessionDuration_Should_Only_Update_Given_Active_Sessions()
+        {
+            using (new TransactionScope())
+            {
+                // Given
+                const int candidateId = 9;
+                var startingSessions = sessionTestHelper.GetCandidateSessions(candidateId);
+
+                // When
+                const int sessionId = 473;
+                sessionDataService.UpdateDelegateSessionDuration(sessionId);
+
+                // Then
+                var updatedSessions = sessionTestHelper.GetCandidateSessions(candidateId).ToList();
+
+                updatedSessions
+                    .Where(session => session.SessionId != sessionId)
+                    .Should()
+                    .BeEquivalentTo(startingSessions.Where(session => session.SessionId != sessionId));
+
+                var activeSession = updatedSessions.First(session => session.SessionId == sessionId);
+                activeSession.LoginTime.AddMinutes(activeSession.Duration)
+                    .Should().BeCloseTo(DateTime.UtcNow, twoMinutesInMilliseconds);
+            }
+        }
+
+        [Test]
+        public void UpdateDelegateSessionDuration_Should_Not_Update_Inactive_Session()
+        {
+            using (new TransactionScope())
+            {
+                // Given
+                const int candidateId = 9;
+                var startingSessions = sessionTestHelper.GetCandidateSessions(candidateId);
+
+                // When
+                const int sessionId = 468;
+                sessionDataService.UpdateDelegateSessionDuration(sessionId);
+
+                // Then
+                var updatedSessions = sessionTestHelper.GetCandidateSessions(candidateId);
+
+                updatedSessions.Should().BeEquivalentTo(startingSessions);
+            }
+        }
+
+        [Test]
+        public void StartAdminSession_Should_Start_Users_New_Admin_Session()
+        {
+            using (new TransactionScope())
+            {
+                // When
+                var adminUser = UserTestHelper.GetDefaultAdminUser();
+                var sessionId = sessionDataService.StartAdminSession(adminUser.Id);
+
+                // Then
+                var newAdminSession = sessionTestHelper.GetAdminSession(sessionId);
+                newAdminSession.Should().NotBe(null);
+                newAdminSession.AdminId.Should().Be(adminUser.Id);
+                newAdminSession.LoginTime.Should().BeCloseTo(DateTime.UtcNow, twoMinutesInMilliseconds);
```

Comment:
Really minor, but I think it should be (just the location of the helping comments):
// Given session id is X
// When GetAdminSession is called with that id
// Then newAdminSession is ... (assertions)